packages/lts-schedule/lib/index.js

```diff
@@ -0,0 +1,317 @@
+/* eslint-disable no-shadow */
+const Fs = require('fs');
+// eslint-disable-next-line import/no-unresolved
+const D3 = require('d3');
+// eslint-disable-next-line import/no-unresolved
+const D3Node = require('d3-node');
+
+const styles = `
+:root {
+  --container: #fffff;
+  --unstable-fill: #ffbe00;
+  --prerelease-fill: #4dd7be;
+  --active-fill: #7652d8;
+  --maintenance-fill: #fd7e40;
+  --text: #161616;
+  --text-on-color: #ffffff;
+  --text-inverse: #ffffff;
+  --border: #e0e0e0;
+}
+
```

Comment:
Spirit design tokens here? 🙂 The light/dark mode queries could then go away.

Reply:
@crishpeen suggested the same. But this was a quick "redesign". I am not sure if this will work with tokens or Sass.